metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/database/DatabaseConfiguration.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.database;
+
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+
+import javax.sql.DataSource;
+
+@Configuration
+public class DatabaseConfiguration {
+
+    @Bean ("db_datasource_properties")
+    @ConfigurationProperties("ena.datasource.db")
+    public DataSourceProperties dbDataSourceProperties() {
+        return new DataSourceProperties();
+    }
+
+    @Bean ("db_datasource")
+    @ConfigurationProperties("ena.datasource.db.hikari")
```

Comment:
This won't work, I've checked the libraries that are pulled with this release train. Using **this version of spring** you don't have a hikari connection pool, so that bit is pointless, as Spring will use the Apache connection pool. This is something they dropped on newer versions and exchanged for Hikari. So either we move up the spring version or we modify this and the properties to configure properly the apache connection pool.

Reply:
Team discussed in slack and a decision on moving to new Spring version is to be made in development meeting